Library/Homebrew/rubocops/text.rb

```diff
@@ -145,15 +151,22 @@ def audit_formula(formula_nodes)
         end
 
         # Check whether value starts with the formula name and then a "/", " " or EOS.
-        def path_starts_with?(path, starts_with)
-          path.match?(%r{^#{Regexp.escape(starts_with)}(/| |$)})
+        # If we're checking for "#{bin}", we also check for "-" since similar binaries also don't need interpolation.
+        def path_starts_with?(path, starts_with, bin = false)
```

Comment:
RuboCop is correct here. I would prefer this as a named arg (which the docs say should work with the node matcher](https://docs.rubocop.org/rubocop-ast/node_pattern.html#param_name-for-named-parameters)) but I can't get it to work.